setup/02-robot_syntax_rule.py

```diff
@@ -1,25 +1,68 @@
-from rflint.common import SuiteRule, TestRule, ERROR
+from rflint.common import SuiteRule, TestRule, ERROR, WARNING
 
 MUST_KEYWORDS = [
         "Open Browser",
         "Input Text",
-        "Page Should Contain"
+        "Input Password"
     ]
 
+LOGIN_KEYWORDS = [
+        "Click Element",
+        "Click Button"
```

Comment:
Should we add `Submit form`? Some webdev-savvy people go for this keyword

Reply:
That is checked in exercise 03. In exercise 02, the participants just replace their "manual steps" with keywords from SeleniumLibrary.

Reply:
Yes, I am referring to the SeleniumLibrary keyword `[Submit Form](https://robotframework.org/SeleniumLibrary/SeleniumLibrary.html#Submit%20Form)` that some webdev-savvy people use :)

Reply:
Ah yes, that should indeed be added.